apps/session-manager/src/utils/api-clients.ts

```diff
@@ -1,21 +1,25 @@
-import { LollipopConfig } from "../config/index";
+import { FastLoginConfig, LollipopConfig } from "../config/index";
 import { FnAppRepo, FnFastLoginRepo, FnLollipopRepo } from "../repositories";
 import { getRequiredENVVar } from "./environment";
 import { httpOrHttpsApiFetch } from "./fetch";
 
 export const initAPIClientsDependencies: () => FnAppRepo.FnAppAPIRepositoryDeps &
   FnFastLoginRepo.FnFastLoginRepositoryDeps &
   FnLollipopRepo.LollipopApiDeps = () => {
-  // Create the API client for the Azure Functions App
+  // Create the API client for `io-functions-app`
   const fnAppAPIClient = FnAppRepo.FnAppAPIClient(
     getRequiredENVVar("API_URL"),
     getRequiredENVVar("API_KEY"),
     httpOrHttpsApiFetch,
   );
+  // Create the API client for `io-functions-fast-login`
   const fnFastLoginAPIClient = FnFastLoginRepo.getFnFastLoginAPIClient(
-    getRequiredENVVar("FAST_LOGIN_API_KEY"),
-    getRequiredENVVar("FAST_LOGIN_API_URL"),
+    FastLoginConfig.FAST_LOGIN_API_KEY,
+    FastLoginConfig.FAST_LOGIN_API_URL,
+    "",
+    httpOrHttpsApiFetch,
```

Comment:
Can we move `API_URL` and `API_URL` env variables too?

Reply:
Fixed in [ddd6e4d](https://github.com/pagopa/io-auth-n-identity-domain/pull/71/commits/ddd6e4d20122bad44a1c648f345ac981eed1b41e)